site/public/js/gradeable.js

```diff
@@ -196,15 +196,33 @@ function renderPeerGradeable(grader_id, gradeable, graded_gradeable, grading_dis
  * @returns {Promise<string>} the html for the graded component
  */
 function renderGradingComponent(grader_id, component, graded_component, grading_disabled, canVerifyGraders, precision, editable, showMarkList, componentVersionConflict) {
+    let gradeable_id = getGradeableId();
+    var custom_mark_req = $.ajax({
+        // Get the TA grading scores	        
+            type: 'GET',       
+            url: buildCourseUrl(['gradeable',gradeable_id, 'getCustomMarksAccess']),
+            dataType: 'html',
+            context: document.body,
+            global: false,
+            async:false,
+            success: function(res) {
+                console.log(res);
+                return res.data;
+            }
+        }).responseText;
+    var custom_mark_obj=JSON.parse(custom_mark_req);
+    var custom_mark_enabled = custom_mark_obj.data;
```

Comment:
Is there a reason you are running this with `async:false` instead of using promises?

Reply:
Nothing particular, just don't have much of experience with promises. Should I update it to use promises instead? @emaicus